src/components/organisms/Room/Participant.tsx

```diff
@@ -0,0 +1,191 @@
+      .map((publication) => publication.track)
+      .filter((track) => track !== null) as AudioTracks;
+
+  useEffect(
+    () =>
+      participant.videoTracks.forEach((x) => {
+        // Video mute handling: for user's own video
+        x?.track?.on("enabled", () => setVideoEnabled(true));
+        x?.track?.on("disabled", () => setVideoEnabled(false));
+      }),
+    [participant.videoTracks]
+  );
+
+  useEffect(() => {
+    setVideoTracks(videoTrackpubsToTracks(participant.videoTracks));
+    setAudioTracks(audioTrackpubsToTracks(participant.audioTracks));
+
+    const trackSubscribed = (track: Track) => {
+      if (track.kind === "video") {
+        // Video mute handling: for other participants' videos
+        track.on("enabled", () => setVideoEnabled(true));
+        track.on("disabled", () => setVideoEnabled(false));
+        setVideoTracks((videoTracks) => [...videoTracks, track]);
+      } else if (track.kind === "audio") {
+        setAudioTracks((audioTracks) => [...audioTracks, track]);
+      }
+    };
+
+    const trackUnsubscribed = (track: Track) => {
+      if (track.kind === "video") {
+        setVideoTracks((videoTracks) => videoTracks.filter((v) => v !== track));
+      } else if (track.kind === "audio") {
+        setAudioTracks((audioTracks) => audioTracks.filter((a) => a !== track));
+      }
+    };
+
+    participant.on("trackSubscribed", trackSubscribed);
+    participant.on("trackUnsubscribed", trackUnsubscribed);
+
+    return () => {
+      setVideoTracks([]);
+      setAudioTracks([]);
+      participant.removeAllListeners();
+    };
+  }, [participant]);
+
+  useEffect(() => {
+    const videoTrack = videoTracks[0];
+    if (videoTrack && videoRef.current) {
+      videoTrack.attach(videoRef.current);
+      return () => {
+        videoTrack.detach();
+      };
+    }
+  }, [videoTracks]);
+
+  useEffect(() => {
+    const audioTrack = audioTracks[0];
+    if (audioTrack && audioRef.current) {
+      audioTrack.attach(audioRef.current);
+      return () => {
+        audioTrack.detach();
+      };
+    }
+  }, [audioTracks]);
+
+  const changeAudioState = useCallback(
+    (enable: boolean) => {
+      if (!enable) {
+        audioTracks?.[0]?.detach();
+      } else {
+        const audioTrack = audioTracks[0];
+        if (audioTrack && audioRef.current) {
+          audioTrack.attach(audioRef.current);
+        }
+      }
+    },
+    [audioTracks]
+  );
+
+  const videoAndAudio = useMemo(
+    () => (
+      <>
+        <video
+          ref={videoRef}
+          autoPlay={true}
+          className={profileData?.mirrorVideo ? "mirrored" : ""}
+        />
+        <audio ref={audioRef} autoPlay={true} />
+      </>
+    ),
+    [profileData]
+  );
+
+  const profileIconClasses = classNames("profile-icon", {
+    "profile-icon__onlyIcon": !videoEnabled,
+    "profile-icon__onlyIcon--remote": !videoEnabled && !isCurrentUser,
+  });
+
+  return (
+    <div className="col participant">
```

Comment:
```suggestion
    <div className="Participant">
```

and in the following class names